Sources/TuistGenerator/Generator/ProjectFileElements.swift

```diff
@@ -454,6 +431,23 @@ class ProjectFileElements {
         elements[fileAbsolutePath] = file
     }
 
+    func addPlayground(from _: AbsolutePath,
+                       fileAbsolutePath: AbsolutePath,
+                       fileRelativePath: RelativePath,
+                       name: String?,
+                       toGroup: PBXGroup,
+                       pbxproj: PBXProj)
+    {
+        let lastKnownFileType = fileAbsolutePath.extension.flatMap { Xcode.filetype(extension: $0) }
+        let file = PBXFileReference(sourceTree: .group,
+                                    name: name,
+                                    lastKnownFileType: lastKnownFileType,
+                                    path: fileRelativePath.pathString)
+        pbxproj.add(object: file)
+        toGroup.children.append(file)
+        elements[fileAbsolutePath] = file
```

Comment:
my apologies @pepibumur 😞

You were originally correct about having the language specifier 

When testing I added a playground manually and saw this in the `.pbxproj` (no language specifier)

```
FBF18BDA2589EC2000AC4338 /* App.playground */ = {isa = PBXFileReference; lastKnownFileType = file.playground; path = App.playground; sourceTree = "<group>"; };
```

However as soon as you open or edit the playground that's when Xcode updates the `.pbxproj` to include the specifier 

```diff
-               FBF18BDA2589EC2000AC4338 /* App.playground */ = {isa = PBXFileReference; lastKnownFileType = file.playground; path = App.playground; sourceTree = "<group>"; };
+               FBF18BDA2589EC2000AC4338 /* App.playground */ = {isa = PBXFileReference; lastKnownFileType = file.playground; path = App.playground; sourceTree = "<group>"; xcLanguageSpecificationIdentifier = xcode.lang.swift; };
```

Reply:
No worries! I added it back. I'll merge when CI passes :)